floating-button-sdk-cafe24.css

```
@@ -528,4 +528,61 @@
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+}
+
+.guest-access-block-md {
+    height: calc(100% - 44px);
+}
+
+.guest-access-block-title {
+    font-family: "pretendard";
```

Stylelint (17.10.0)

[error] Expected no quotes around "pretendard" (font-family-name-quotes)

(font-family-name-quotes)